compiler/src/Language/Bond/Codegen/Java/Class_java.hs

```diff
@@ -235,13 +235,16 @@ makeStructBondTypeMember_StructBondTypeBuilderImpl declName declParams = [lt|
 
             @Override
             protected final #{typeDescriptorName declName} buildNewInstance(org.bondlib.BondType[] genericTypeArguments) {
-                return new StructBondTypeImpl(#{ifThenElse (null declParams) "null" "new org.bondlib.GenericTypeSpecialization(genericTypeArguments)"});
+                return new StructBondTypeImpl(#{ifThenElse (null declParams) nullStr "new org.bondlib.GenericTypeSpecialization(genericTypeArguments)"});
             }
 
             static void register() {
                 registerStructType(#{declName}.class, new StructBondTypeBuilderImpl());
             }
         }|]
+    where
+        nullStr :: Text
```

Comment:
`nullText`

Reply:
You should be able to shorten this to one line:

```
nullText = "null" :: Text
```